addons/l10n_ec_ats/account_invoice/factura_retencion.py

```diff
@@ -0,0 +1,70 @@
+# -*- coding: utf-8 -*-
+from openerp import models, fields, api
+
+class account_invoice(models.Model):
+    _inherit = 'account.invoice'
+# Computo del IVA para la impresion QWeb
+    x_impuesto = fields.Monetary(string='Impuesto', store=True, readonly=True, compute='_compute_impuesto')
+# Secuencial del documento Impreso Company
+    x_company_establecimiento = fields.Char(string='Mi Establecimiento', store=True, readonly=True, compute='_compute_establecimiento',\
```

Comment:
Why do you need to start your fields with x_, that's a way Odoo creates fields when you do it via web, but it's not a requirement on code.